openmdao/core/system.py

```diff
@@ -1775,6 +1775,11 @@ def update_src_indices(name, key):
                 abs_name = self._var_allprocs_prom2abs_list['input'][name][0]
                 meta = self._var_abs2meta[abs_name]
 
+                if isinstance(src_indices, tuple):
```

Comment:
I can't see where 'indices' is used anywhere in this function.  I think this whole conditional block may be unnecessary.